packages/hdwallet-keepkey/src/bitcoin.ts

```diff
@@ -70,36 +70,7 @@ function prepareSignTx(
     const output: core.BTCSignTxOutput = o;
     const newOutput = new Types.TxOutputType();
     newOutput.setAmount(Number(output.amount));
-    if (output.exchangeType) {
```

Comment:
seems like unrelated changes. was this related to legacy shapeshift exchange?

Reply:
if not supported anymore, I would clean up any related types as well and just make a separate PR for this removal